web/wp-config.php

```diff
@@ -14,180 +14,75 @@
  */
 require_once( $rootPath . '/vendor/autoload.php' );
```

Comment:
I see that this one addition at least is needed over what is in the standard pantheon-systems/wordpress repo.